static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -13,20 +12,16 @@ export const show = (forceModal: ?boolean): Promise<boolean> => {
     } else {
         require.ensure(
```

Comment:
We probably could get rid of this now that we're not requiring `common/module/cmp-ui`.

Reply:
does this whole file still serve a purpose?

Reply:
Sign-in gates uses a lot of functions from it. We can remove it in a subsequent PR.